try 
{
    let x = int(null);
}
catch(e)
{
    print "Caught error! '" + e + "'.";
}